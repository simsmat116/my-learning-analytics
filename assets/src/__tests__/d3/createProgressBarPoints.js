import createProgressBar from '../../components/d3/createProgressBar'
import { assignmentData } from '../testData/assignmentProgressBarData'

describe('createProgressBar', () => {
  test('should build a Progress Bar using points data and correctly show points and not percentages', () => {
    const div = document.createElement('div')
    createProgressBar({
      data: assignmentData.progress,
      gradeType: 'PT',
      totalPoints: assignmentData.total_points,
      aspectRatio: 0.12
    })
    expect(div).toMatchSnapshot()
  })

})
