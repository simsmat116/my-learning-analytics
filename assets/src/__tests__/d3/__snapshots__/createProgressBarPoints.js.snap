// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createProgressBar should build a Progress Bar using points data and correctly show points and not percentages 1`] = `<div />`;
